parenthetics.py

```diff
@@ -0,0 +1,16 @@
+"""parenthetics code-kata."""
+
+
+def parenthetics(string):
+    """Return 1, 0, or -1 depending on the string input."""
+    queue = []
```

Comment:
this is a list, not a queue. This is not one of the data structures that you built this week.